jp/event_summaries_api.md

```diff
@@ -35,57 +35,54 @@ Events that are shown on the dashboard cover a number of different types.  For e
 | 'task'     | /api/tasks/567 |
 | 'basic'    | /api/basic_events/456|
 
-It is likely useful to include a bit more information about each event so that the frontend have a more useful display without making one extra GET for each event.  E.g. if the frontend knew that the plan was an iReading plan it could style it differently than a homework plan.  Also some events may have type-specific data like a "trouble" warning for teachers on iReading plans to indicate that students are not faring well in their reading assignments.
+It is likely useful to include a bit more information about each event so that the frontend have a more useful display without making one extra `GET` for each event.  E.g. if the frontend knew that the plan was an iReading plan it could style it differently than a homework plan.  Also some events may have type-specific data like a "trouble" warning for teachers on iReading plans to indicate that students are not faring well in their reading assignments.
 
 We could encode this extra information with nested data structures that contain a subset of the info that would be retrieved from calling the event type's particular endpoint, e.g.:
 
-```ruby
+```coffee
 {
   events: [
     {
-      title: 'Chapter 2 Reading',
-      starts: 20140202-08:08:01,
-      ends: 20140202-08:08:01,
-      type: 'plan',
-      id: 42,
-      data: {  # would contain selected info from GET /api/plans/42
-        type: 'reading',
-        trouble: false,
+      start_at: '20140202-08:08:01'
+      end_at: '20140202-08:08:01'
+      payload_type: 'plan'
+      payload:  # would contain selected info from GET /api/plans/42
+        id: 42
+        type: 'reading'
+        title: 'Chapter 2 Reading'
+        is_trouble: false
         stats: { # maybe include, maybe not
         }
-      }
-    },
+    }
     {
-      title: 'Chapter 2 Reading',
-      starts: 20140202-08:08:01,
-      ends: 20140202-08:08:01,
-      type: 'task',
-      id: 87,
-      data: { # would contain selected info from GET /api/tasks/87
-        type: 'homework',
-        complete: false
-      }
-    },
+      start_at: '20140202-08:08:01'
+      end_at: '20140202-08:08:01'
+      payload_type: 'task'
```

Comment:
The concatenated approach has some issues tho (the payload wouldn't look quite like what you'd get from `/api/plans/42`).  Another alternative would be to replace `payload_type` with `payload_endpoint: '/api/plans/42'`, but maybe that isn't great for you because then you have to pull out "plans" out of that.  Just brainstorming...